### PR TITLE
Add labels in openshift route resource

### LINKED ISSUE #336: Missing custom labels on route in litmus chart
Hello,

There is no possibility to add customLabel to the route when using openshift that can cause issue during deployment. Also commons issue are not applied.

Global customLabel are also not applied in this resource.
I will open a PR for this fix.

Have a nice day

### PR DESCRIPTION
**Pull request by Calvinaud**
- Add common and custom labels to the route resource
- Upgrade chart version to 2.15.12
- Regenerate doc with helm-docs

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #336

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] DCO signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: charts/litmus/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # litmus
 
-![Version: 3.0.0](https://img.shields.io/badge/Version-3.0.0-informational?style=flat-square) ![AppVersion: 3.0.0](https://img.shields.io/badge/AppVersion-3.0.0-informational?style=flat-square)
+![Version: 3.0.1](https://img.shields.io/badge/Version-3.0.0-informational?style=flat-square) ![AppVersion: 3.0.0](https://img.shields.io/badge/AppVersion-3.0.0-informational?style=flat-square)
```
Comment (gdsoumya) [inline charts/litmus/README.md:3]: Should this be?

```suggestion
![Version: 3.0.1](https://img.shields.io/badge/Version-3.0.1-informational?style=flat-square) ![AppVersion: 3.0.0](https://img.shields.io/badge/AppVersion-3.0.0-informational?style=flat-square)
```

**Review by gdsoumya — CHANGES_REQUESTED** [review]

**Review by Jasstkn — APPROVED** [review]

File: charts/litmus/values.yaml [hunk 2]
```diff
@@ -325,6 +325,7 @@ openshift:
   route:
     enabled: false
     name: litmus-portal
+    customLabels: {}
```
Comment (Jasstkn) [inline charts/litmus/values.yaml:328]: I prefer `additionalLabels` more but if others are fine, then i doesn't matter much. approving for now.

**Review by gdsoumya — APPROVED** [review]

File: charts/litmus/values.yaml (hunk 2 above)
Comment (Calvinaud) [reply charts/litmus/values.yaml:328]: In the values.yaml all others resource are using `customLabels` (https://github.com/litmuschaos/litmus-helm/blob/master/charts/litmus/values.yaml#L10) that why I used the same name.

Comment (gdsoumya) [reply charts/litmus/values.yaml:328]: It's probably okay then.

**Pull request merged**